content/en/docs/setup/learning-environment/minikube.md

```diff
@@ -350,6 +350,12 @@ To access the [Kubernetes Dashboard](/docs/tasks/access-application-cluster/web-
 minikube dashboard
 ```
 
+Under the hood above command proxies internal cluster resources onto local host and appends dashboard API path:
```

Comment:
@dzmitry-lahoda I'm afraid I don't understand what the reader should learn from reading this. Can you clarify?

Reply:
it is not clear why `minikube dashboard` hang after run and what it really does. But things become clear when read about `proxy`. same time user gets command which will work not only on minikube, but on production environment either (i hope).

Reply:
@dzmitry-lahoda, According to Style guide (https://kubernetes.io/docs/contribute/style/style-guide/#avoid-jargon-and-idioms), using a jargon is discouraged. Can you update the sentence accordingly?